scripts/linters/pylint_extensions.py

```diff
@@ -879,6 +928,7 @@ def check_docstring_section_indentation(self, node):
             args_indentation_in_spaces = 0
             docstring = node.doc.splitlines()
             self.check_newline_above_args(node, docstring)
+            self.check_newline_below_docstring_sections(node, docstring)
```

Comment:
Why is this check added inside `check_docstring_section_indentation`? Should it be inside `check_docstring_structure` or just as an independent check? (not sure) Let me know what you think?

Reply:
I was following the flow of the `check_newline_above_args` test, which is actually the check for newlines above docstring sections (I think this check might need to be renamed).